README.md

```diff
@@ -1,34 +1,35 @@
+OpenSearch Benchmark is the macrobenchmarking framework for OpenSearch.
 
-What is Benchmark?
-------------------
+What is OpenSearch Benchmark?
+-----------------------------
 
-You want to benchmark OpenSearch? Then Benchmark is for you. It can help you with the following tasks:
+If you are looking to performance test OpenSearch, then OpenSearch Benchmark is for you. It can help you with the following tasks:
 
-* Setup and teardown of an OpenSearch cluster for benchmarking
-* Management of benchmark data and specifications even across OpenSearch versions
-* Running benchmarks and recording results
-* Finding performance problems by attaching so-called telemetry devices
+* Running performance benchmarks and recording results
+* Setting up and tearing down OpenSearch clusters for benchmarking
+* Managing benchmark data and specifications across OpenSearch versions
+* Discovering performance problems by attaching so-called telemetry devices
 * Comparing performance results
+* Creating customized workloads
 
-We have also put considerable effort in Benchmark to ensure that benchmarking data are reproducible.
+We have also put considerable effort in OpenSearch Benchmark to ensure that benchmarking data are reproducible.
 
 Quick Start
 -----------
 
-Benchmark is developed for Unix and is actively tested on Linux and MacOS. Benchmark supports [benchmarking OpenSearch clusters running on Windows](<https://github.com/opensearch-project/OpenSearch-Benchmark/blob/main/DEVELOPER_GUIDE.md>) but Benchmark itself needs to be installed on machines running Unix.
+OpenSearch Benchmark is developed for Unix and is actively tested on Linux, Ubuntu, and MacOS. OpenSearch Benchmark supports [benchmarking OpenSearch clusters running on Windows](<https://github.com/opensearch-project/OpenSearch-Benchmark/blob/main/DEVELOPER_GUIDE.md>) but OpenSearch Benchmark itself needs to be installed on machines running Unix.
 
-Installing Benchmark
---------------------------
+Installing OpenSearch Benchmark
+-------------------------------
 
-**Note**: If you actively develop on OpenSearch, we recommend that you [install Benchmark in development mode](<https://github.com/opensearch-project/OpenSearch-Benchmark/blob/main/DEVELOPER_GUIDE.md>) instead as OpenSearch is fast moving and Benchmark always adapts accordingly to the latest main version.
+**Note**: If you actively develop on OpenSearch, we recommend that you [install OpenSearch Benchmark in development mode](<https://github.com/opensearch-project/OpenSearch-Benchmark/blob/main/DEVELOPER_GUIDE.md>) instead as OpenSearch is fast moving and OpenSearch Benchmark always adapts accordingly to the latest main version.
 
 Install Python 3.8+ including ``pip3``, git 1.9+ and an [appropriate JDK to run OpenSearch](<https://github.com/opensearch-project/OpenSearch-Benchmark/blob/main/DEVELOPER_GUIDE.md>) Be sure that ``JAVA_HOME`` points to that JDK. Then run the following command, optionally prefixed by ``sudo`` if necessary:
```

Comment:
points to that JDK and that the corresponding `java` executable in `$JAVA_HOME/bin` is picked up via your PATH environment variable.